package.json

```diff
@@ -178,6 +178,7 @@
   "packageManager": "yarn@3.0.2",
   "resolutions": {
     "terser": "~4.8.1",
-    "d3-color": "~3.1.0"
+    "d3-color": "~3.1.0",
+    "nwsapi": "^2.2.1"
```

Comment:
I was original going to suggest adding a comment here with a link to https://security.snyk.io/vuln/SNYK-JS-NWSAPI-2841516, but we can't do JSON comments.  Can you instead modify the commit message to reference that link as the reason why we are making this change?  I know it's in the OP in GitHub, but commits are forever, and also work with `git blame`